torch/_inductor/codegen/common.py

```diff
@@ -733,7 +735,10 @@ class CSEProxy:
             def __getattr__(name):
                 def inner(*args, **kwargs):
                     csevar = self.cse.generate(
-                        self.compute, getattr(parent_handler, name)(*args, **kwargs)
+                        self.compute,
+                        getattr(parent_handler, name)(*args, **kwargs),
+                        origin_args=args,
```

Comment:
these changes are no longer needed?

Reply:
Thanks! fixed in https://github.com/pytorch/pytorch/pull/101468/commits/7b9d59c4ed675654d476993ab1d858a3d99f5b45